test/remote/gateways/remote_stripe_emv_test.rb

```diff
@@ -131,20 +127,19 @@ def test_authorization_and_void_with_emv_credit_card_in_uk
   end
 
   def test_authorization_and_void_with_emv_contactless_credit_card
```

Comment:
is `authorization` a transaction type that's supported for emv contactless?  does it get captured automatically?

Reply:
It will get captured automatically but you're right, this test doesn't match how we use the gateways. I'll fix it.